examples/md/main.ts

```diff
@@ -0,0 +1,13 @@
+import { dynamicRequireShowdown } from "https://etok.codes/dynamic_require/raw/main/examples/showdown/mod.ts";
```

Comment:
To prevent cross-site scripting hurdles, let's omit `dynamicRequireShowdown`.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/194339961caddbcd7a91c8428ace324a2cd5f209>.